src/Components/ViewProject/ViewProject.tsx

```diff
@@ -1,7 +1,75 @@
-import React from 'react'
+import React, { useContext, useEffect, useState } from 'react'
+import avatar from '../../assets/avatar.png'
+import axios from 'axios'
+import { AuthContext } from '../../Context/AuthContext'
+import { useParams } from 'react-router-dom'
 
 export default function ViewProject() {
-  return (
-    <div>ViewProject</div>
+  const{id}=useParams()
+  const [viewPro,setViewPro]=useState([])
+const {baseUrl,reqHeaders}:any=useContext(AuthContext)
+useEffect(()=>{
+  axios.get(`${baseUrl}/Project/${id}`,{headers:reqHeaders}).then((response)=>{
+    console.log(response);
+    setViewPro(response?.data?.data)
```

Comment:
Did you test that?
the response coming from the server has only one nested "data", and it's not a array it's an object.
So, it should be as the following [response?.data]